test/MorseL.Scaleout.Redis.Tests/EndToEndTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using MorseL.Shared.Tests;
+using StackExchange.Redis;
+using Xunit;
+
+namespace MorseL.Scaleout.Redis.Tests
+{
+    public class EndToEndTests
+    {
+        [Fact]
+        public async void ConnectionSubscriptionAddedOnConnect()
+        {
+            RedisBackplane backplane = null;
+            using (new SimpleMorseLServer<TestHub>(IPAddress.Any, 5000, (collection, builder) =>
```

Comment:
We should probably look at doing something like the builder pattern to reduce the setup noise in these tests (please, please don't just move it into a setup function 😿 )